### DISCUSSION TITLE
Error reading scdocx file

### DISCUSSION THREAD

**Discussion by waltersma** [ansys/pyprimemesh#904] [Q&A]
Original asked in https://github.com/ansys/pyprimemesh/discussions/901#discussioncomment-10809280

After I installed the 0.5.1 Version of the client, Another Error occured, which didn't before:

Traceback (most recent call last):
File "c:\Users\heinr\Documents\Masterthesis-local\UnitCellAnalyser\test.py", line 15, in
mesh_util.read(file_name=".\Working_dir\Fluid_model.scdocx")
File "C:\Users\heinr\AppData\Roaming\Python\Python312\site-packages\ansys\meshing\prime\lucid\mesh_util.py", line 92, in read
prime.FileIO(self._model).import_cad(
File "C:\Users\heinr\AppData\Roaming\Python\Python312\site-packages\ansys\meshing\prime\core\fileio.py", line 504, in import_cad
import_result = super().import_cad(temp_file_name, params)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "C:\Users\heinr\AppData\Roaming\Python\Python312\site-packages\ansys\meshing\prime\autogen\fileio.py", line 650, in import_cad
result = self._comm.serve(self._model, command_name, self._object_id, args=args)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "C:\Users\heinr\AppData\Roaming\Python\Python312\site-packages\ansys\meshing\prime\internals\error_handling.py", line 535, in wrapper_error_code
raise PrimeRuntimeError(
ansys.meshing.prime.internals.error_handling.PrimeRuntimeError: CAD import failed.

I checked the Path of my CAD-File and also tried to import a step file. What could be the issue here?

**Comment by waltersma** [comment]
Hi @stenghe

A few comments and things to try:

- Have you setup the CAD configuration for your 24R1 installation?
- Can you see if the scdocx file that you are reading can be read into the 24R1 build of SpaceClaim you have installed on the machine?
- In your script can you try the absolute path as a raw string to make sure it is not an issue.
- Are you using a virtual environment?
- Please note that at 0.5.1 python version 3.12 is not a supported version (supported versions are 3.8 to 3.11).

Hope these comments help. Thanks

**Reply by stenghe** [reply]
Thanks a lot for your suggestions! I'm using a virtual environment. I changed the python version to 3.11.9 now, but the error persists. I also try the rest of your points. Any other ideas? Thanks a lot!

**Reply by aglowinski-danfoss** [reply]
Hello, may I join your discussion?
I am actually facing the exact same isse. I am running Ansys 24R2 locally on a VDI and cannot import a Discovery File into PyPrimeMesh anymore. The same script works on several desktop PCs with 24R2. I am also getting the error:
import_result = super().import_cad(temp_file_name, params)
ansys.meshing.prime.internals.error_handling.PrimeRuntimeError: CAD import failed.
Any ideas?
Best regards,
Alex

**Reply by aglowinski-danfoss** [reply]
For comparison I tried one of the examples:
https://prime.docs.pyansys.com/version/stable/examples/gallery_examples/gallery/07_saddle_bracket.html#sphx-glr-examples-gallery-examples-gallery-07-saddle-bracket-py
But this gives the same error for the CAD import.
